lib/ProductOpener/KnowledgePanels.pm

```diff
@@ -598,7 +598,7 @@ sub create_ecoscore_panel($$$) {
             # Add properties of interest
             foreach my $property (qw(environmental_benefits description)) {
                 my $property_value = get_inherited_property("labels", $labelid, $property . ":" . $target_lc);
-                if (not defined $property_value && $target_lc != "en") {
+                if (!(defined $property_value) && ($target_lc != "en")) {
```

Comment:
I know it was like this before, but ($target_lc != "en") is a bug, it should be ($target_lc ne "en"), as != is for numeric comparison, not string comparison.

e.g. try this:

perl -e "if ('fr' != 'en') { print 'never reached'}"

Reply:
How does this look?